scripts/api/data/licenses/licenseApache-2.0.json

```diff
@@ -0,0 +1,10 @@
+{
+  "rightsName": "Apache 2.0",
```

Comment:
In the docs, we should add all these new licenses to the list at https://github.com/philippconzett/dataverse/blob/8512-standardize-license-configuration/doc/sphinx-guides/source/installation/config.rst#adding-licenses so they are easily downloadable to installations that would like to use them.